model/src/pyrenew/model/admissionsmodel.py

```diff
@@ -208,7 +208,7 @@ def sample(
                     observed_hosp_admissions,
                     *_,
                 ) = self.hosp_admission_obs_process_rv.sample(
-                    mu=latent_hosp_admissions,
+                    mu=latent_hosp_admissions[i0_size + padding :],
```

Comment:
Mostly a note to myself: I actually think we can remove the entire if-else portion of the model.